### ISSUE TITLE
edk2 build fails

### ISSUE THREAD

**Issue by shorie000** [sarisia/mikanos-docker#11]
edk2 build fails.

Here is the error log
```
------
 > [5/7] RUN git clone --recursive https://github.com/tianocore/edk2.git edk2  && (cd edk2 && git checkout 38c8be123aced4cc8ad5c7e0da9121a181b94251)  && make -C edk2/BaseTools/Source/C:
#8 114.2 /usr/bin/ld: /home/vscode/edk2/BaseTools/Source/C/BrotliCompress/brotli/c/enc/fast_log.h:53: undefined reference to `kBrotliLog2Table'
#8 114.2 /usr/bin/ld: /home/vscode/edk2/BaseTools/Source/C/BrotliCompress/brotli/c/enc/fast_log.h:53: undefined reference to `kBrotliLog2Table'
#8 114.2 /usr/bin/ld: brotli/c/enc/metablock.o:/home/vscode/edk2/BaseTools/Source/C/BrotliCompress/brotli/c/enc/fast_log.h:53: more undefined references to `kBrotliLog2Table' follow
#8 114.2 /usr/bin/ld: ../bin/BrotliCompress: hidden symbol `kBrotliCopyExtra' isn't defined
#8 114.2 /usr/bin/ld: final link failed: bad value
#8 114.2 collect2: error: ld returned 1 exit status
#8 114.2 make[1]: *** [../Makefiles/app.makefile:18: ../bin/BrotliCompress] Error 1
#8 114.2 make[1]: Leaving directory '/home/vscode/edk2/BaseTools/Source/C/BrotliCompress'
#8 114.2 make: Leaving directory '/home/vscode/edk2/BaseTools/Source/C'
#8 114.2 make: *** [GNUmakefile:76: BrotliCompress] Error 2
------
error: failed to solve: executor failed running [/bin/sh -c git clone --recursive https://github.com/tianocore/edk2.git edk2  && (cd edk2 && git checkout 38c8be123aced4cc8ad5c7e0da9121a181b94251)  && make -C edk2/BaseTools/Source/C]: exit code: 2
Error: Process completed with exit code 1.
```

**Comment by sarisia** [comment]
Thanks, I’ m looking into this.

**Comment by sarisia** [comment]
This is fixed in #12. Thank you for reporting!

**Comment by shorie000** [comment]
Thank you for dealing with it quickly!
